lib/minitest/assertions.rb

```diff
@@ -564,7 +572,7 @@ def refute test, msg = nil
     end
 
     ##
-    # Fails if +obj+ is empty.
+    # Fails if +obj+ is not empty.
```

Comment:
Not quite. This is either "Fails if obj is empty" or "Fails unless obj is not empty".

Think of it backwards: it's the opposite of `assert_empty` which "fails if obj is non-empty".

Reply:
Right, I don't know what I was thinking :/. Thanks for getting this in.